apps/emqx_cluster_link/src/emqx_cluster_link_schema.erl

```diff
@@ -0,0 +1,154 @@
+%%--------------------------------------------------------------------
+%% Copyright (c) 2024 EMQ Technologies Co., Ltd. All Rights Reserved.
+%%--------------------------------------------------------------------
+
+-module(emqx_cluster_link_schema).
+
+-behaviour(emqx_schema_hooks).
+
+-include("emqx_cluster_link.hrl").
+-include_lib("hocon/include/hoconsc.hrl").
+
+-export([injected_fields/0]).
+
+%% Used in emqx_cluster_link_api
+-export([links_schema/1]).
+
+-export([
+    roots/0,
+    fields/1,
+    namespace/0,
+    desc/1
+]).
+
+-import(emqx_schema, [mk_duration/2]).
+
+-define(MQTT_HOST_OPTS, #{default_port => 1883}).
+
+namespace() -> "cluster".
+
+roots() -> [].
+
+injected_fields() ->
+    #{cluster => [{links, links_schema(#{})}]}.
+
+links_schema(Meta) ->
+    ?HOCON(?ARRAY(?R_REF("link")), Meta#{
+        default => [], validator => fun links_validator/1, desc => ?DESC("links")
+    }).
+
+fields("link") ->
+    [
+        {enable, ?HOCON(boolean(), #{default => true, desc => ?DESC(enable)})},
+        {name, ?HOCON(binary(), #{required => true, desc => ?DESC(link_name)})},
+        {server,
+            emqx_schema:servers_sc(#{required => true, desc => ?DESC(server)}, ?MQTT_HOST_OPTS)},
+        {clientid, ?HOCON(binary(), #{desc => ?DESC(clientid)})},
+        {username, ?HOCON(binary(), #{desc => ?DESC(username)})},
+        {password, emqx_schema_secret:mk(#{desc => ?DESC(password)})},
+        {ssl, #{
+            type => ?R_REF(emqx_schema, "ssl_client_opts"),
+            default => #{<<"enable">> => false},
+            desc => ?DESC(ssl)
+        }},
+        {topics,
+            ?HOCON(?ARRAY(binary()), #{
+                desc => ?DESC(topics), required => true, validator => fun topics_validator/1
+            })},
```

Comment:
```suggestion
            ?HOCON(?ARRAY(binary()), #{
                desc => ?DESC(topics), default => [<<"#">>], validator => fun topics_validator/1
            })},
```

Reply:
No, `[]` means this cluster wants to receive no external messages, but it is ready to forward its messages to the link if requested. It can be used to set-up unidirectional links.

Reply:
given that topics can be changed incrementally, this default is ok.